plugins/jobs/web_client/templates/JobList.pug

```diff
@@ -34,5 +34,3 @@ table.g-jobs-list-table
           td= job.get('type')
         if columns & columnEnum.COLUMN_STATUS
           td.g-job-status-cell= JobStatus.text(job.get('status'))
-
-.g-job-pagination
```

Comment:
This file has been basically split into two files. So, I don't think any code were using it could use it without change.